### PR TITLE
add icon

### PR DESCRIPTION
**Pull request by HassanFdim**
I added a icon to the website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,13 +2,11 @@
 <html lang="en">
 <head>
```
Comment (Abdallah01win) [inline index.html:3]: Where is the </head> closing tag?
close the head before opening the body.

File: index.html [hunk 2]
```diff
@@ -2,13 +2,11 @@
...
 
-<body>
 
-<a target="_blank"href="https://en.wikipedia.org/wiki/Nature">
-<video src="video1.mp4" width="500px" controls autoplay muted loop>
-  <source src="video1.mp4" type="video/mp4">
-  <source src="video1.webm" type="video/webm">
+<link rel="icon" type="image/jpg" href="favicon.jpg">
+<title>My website </title>
```
Comment (Abdallah01win) [inline index.html]: The link and title should be inside the <head> tag

**Review by Abdallah01win — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (HassanFdim) [reply index.html]: Ohh okay I will

**Review by Abdallah01win — APPROVED** [review]
That's good

**Pull request merged**